drivers/pds/pdsbackup/backup_utils.go

```diff
@@ -21,6 +21,7 @@ const (
 	bkpTimeInterval      = 60 * time.Second
 	bkpMaxtimeInterval   = 10 * time.Minute
 	BACKUP_JOB_SUCCEEDED = "Succeeded"
+	minioEndpoint        = "minio.pwx.dev.purestorage.com"
```

Comment:
Instead of hardcoding the endpoint can we fetch this either from jenkins pipeline variables or any other config map.

Reply:
agreed, since we are hardcoding the s3 point also I thought of following the same style.
also we are not using any other minio endpoints for testing, shall we keep this for now? because changing this needs good amount of rework on test file also on the jenkins file.

Reply:
Sure

Reply:
I have fixed it in the same PR, since we are getting new minio endpoints.